workspaces/ui-v2/src/pages/changelog/ChangelogEndpointRootPage.tsx

```diff
@@ -19,6 +18,7 @@ import {
 import { useEndpointBody } from '<src>/hooks/useEndpointBodyHook';
 import { SubtleBlueBackground } from '<src>/constants/theme';
 import { CodeBlock, EndpointTOC, TwoColumn } from '<src>/pages/docs/components';
+import { useAppSelector } from '<src>/store';
```

Comment:
Just to verify for my own sanity: this to use a selector on the app-level-state (Redux store), right? Read a bit as if a different app can be selected 😅.

Reply:
Correct! It's a reexport of useSelector but with the typing of the current app's store typing (otherwise we cannot use typescript safety checks). Alternative names are welcome